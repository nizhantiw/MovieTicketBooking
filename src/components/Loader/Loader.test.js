import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer'
import Loader from './Loader'

import { render, screen } from '@testing-library/react';

configure({adapter: new Adapter()});

configure({adapter: new Adapter()});
it("renders correctly", () => {
  const wrapper = shallow(
    <Loader />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders a snapshot', () => {
  const tree = renderer.create(<Loader/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('should show the text', () => {
    const toggleInstance = shallow(<Loader />);
    const element = toggleInstance.find('div span');
    expect(element.text()).toMatchSnapshot();
   });


test('render h1 element', () => {
  render(<Loader />);
  expect(screen.getByText('Loading...')).toBeInTheDocument();
});