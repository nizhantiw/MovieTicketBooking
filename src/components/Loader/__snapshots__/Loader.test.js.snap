// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="spinner-border"
  role="status"
  style={
    Object {
      "height": "4rem",
      "width": "4rem",
    }
  }
>
  <span
    className="sr-only"
  >
    Loading...
  </span>
</div>
`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;

exports[`should show the text 1`] = `"Loading..."`;
